### ISSUE TITLE
aria-label on icons

### ISSUE THREAD

**Issue by alfasou** [validator/validator#1409]
It seems to be reporting aria-label on icons as a misuse.

**Comment by scottaohara** [comment]
Depending on the html element you are using to render the icon, it may very well be a misuse, as aria-label is only supposed to be used on html elements with implicit or explicit aria roles that allow the attribute

**Comment by sideshowbarker** [comment]
Closing for lack of necessary details.

@alfasou If you can provide the details in https://github.com/validator/validator/issues/new?assignees=&labels=bug&template=incorrect-validation.md&title= we could reopen this.

**Comment by lukaszmn** [comment]
@alfasou I think you missed the `role="img"` attribute in your icon, as https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24 suggests.
